src/main/java/io/cdap/e2e/pages/locators/CdfNameSpaceAdminLocators.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Represents CdfNameSpaceAdminLocators
+ */
+public class CdfNameSpaceAdminLocators {
+
+  @FindBy(how = How.XPATH, using = "//div[contains(text(),'Namespace Admin')]")
```

Comment:
Finding elements using text / copy is generally not a good idea, as copy may change based on product requirements and this may result in breaking tests across multiple projects (as this locator is in the e2e framework). Prefer using a `data-testid` for locating elements.